homework/shelkovich_ilya/primitiveAndWrapperSpeed/src/test/java/com/tutrit/java/quickstart/service/SumOfPrimitiveAndWrapperTest.java

```diff
@@ -0,0 +1,37 @@
+package com.tutrit.java.quickstart.service;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class SumOfPrimitiveAndWrapperTest {
+
+    Sum sum;
+    int a;
+    int b;
+    Integer c;
+    Integer d;
+
+    @Before
+    public void setUp() {
+        this.sum = new Sum();
+        this.a = 10;
+        this.b = 10;
+        this.c = 10;
+        this.d = 10;
+    }
+
+    @Test
+    public void cycleSumOfPrimitives() {
+        int expected = 10_000_000;
+        int actual = sum.cycleSumOfPrimitives();
+        Assert.assertEquals(expected, actual);
+    }
+
+    @Test
+    public void cycleSumOfWrapper() {
+        int expected = 10_000_000;
+        int actual = sum.cycleSumOfWrapper();
+        Assert.assertEquals(expected, actual);
+    }
```

Comment:
well, these test just verifying that cycle had 10_000_000 iterations. 
Please, take a look on @timeout annotation. 

the reasonable case is to compare time spent on primitive vs reference calculations, what means that sum.cycleSumOfWrapper() should return time spent.
and, obviously, renamed like millsToCalculateWrapers()
Another method should be modified same way. After that, your assertion'll be like 
assetTrue(miilsToCalculateWrappers() > millsToCalculatePrimitives);

Reply:
improved it